### DISCUSSION TITLE
useController TypeScript Error

### DISCUSSION THREAD

**Discussion by MdImthiyas** [react-hook-form/react-hook-form#4429] [Q&A]
Hi folks, I'm getting the below error in useController hooks while running the Unit Test Cases using @testing-library/react.

**Error Message:**: Controller is missing `control` prop.

CodeSandbox link: [https://codesandbox.io/s/usecontroller-forked-hzon6?file=/src/App.tsx](https://codesandbox.io/s/usecontroller-forked-hzon6?file=/src/App.tsx)

Kindly help me to solve this issue.

Thanks!

**Comment by bluebill1049** [comment]
you can take a look at this https://github.com/react-hook-form/react-hook-form/blob/v7/src/__tests__/useController.test.tsx maybe it helps.

**Reply by MdImthiyas** [reply]
@bluebill1049 , Thank you will look into that.
